api/src/main/java/org/pmiops/workbench/access/AccessUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.access;
+
+import java.util.List;
+import java.util.Optional;
+import org.pmiops.workbench.db.model.DbAccessTier;
+
+/** Utilities for RW Access related functionalities. */
+public class AccessUtils {
+  private AccessUtils() {}
+
+  /** Returns the {@link DbAccessTier} from list of access tiers by tier short name. */
+  public static Optional<DbAccessTier> getAccessTierByShortName(
+      List<DbAccessTier> accessTierList, String accessTierShortName) {
```

Comment:
Create a static util because I want to avoid unnecessary loading and injecting the 'heavy' AccessTierService.